app/helpers/application_helper.rb

```diff
@@ -1,2 +1,14 @@
 module ApplicationHelper
+  def markdown(text)
```

Comment:
@vipul-vinsol  use basic presenter gem to make question presenter and do like this question.presenter.content